### DISCUSSION TITLE
PWA login page back button missing

### DISCUSSION THREAD

**Discussion by turanibrahim** [kimai/kimai#3975] [General]
I installed Kimai as PWA application in my Mac. It works perfect, the new UI is amazing, but whenever I need to do that requires password it shows me the page below. This page doesn't have a back button, so I have to close and reopen PWA because of it. It would be better to add back button top of page for PWA users.
<img width="1503" alt="Kimai_Time-Tracker_-_Kimai_–_Time_Tracking_and_browser_back_shortcut_mac_-_Google_da_Ara" src="https://user-images.githubusercontent.com/25150528/232333709-a81ca7e8-e13e-4922-9a67-68016bb03de5.png">

**Comment by kevinpapst** [comment]
I don't understand the problem. 
Why do you close the PWA on these screens and don't use your password?

You can configure the REMEMBER ME time currently only here:
https://github.com/kimai/kimai/blob/main/config/packages/security.yaml#L43

Does that screen show up more often? I have no real experience with PWA, so maybe there is some tuning needed, which I am not aware of. 
The back button sounds like a weird workaround ... do you know how to show it only in PWA context?

**Reply by turanibrahim** [reply]
I don't want to continue on this page, I just misclicked on the button, so I want to go back, but there is no button to do it. So, it would be better to have back button like browsers have because in PWA mode there is no back button. Just small button that trigger `window.history.back()` function. I couldn't find the page in the repo, so I couldn't do it myself, sorry.
<img width="1502" alt="Kimai_Time-Tracker_-_Kimai_–_Time_Tracking" src="https://user-images.githubusercontent.com/25150528/233775653-b0a2392f-ca4b-4c18-b279-eb6182cd0d3d.png">

**Reply by kevinpapst** [reply]
> do you know how to show it only in PWA context?

**Reply by turanibrahim** [reply]
```scss
.visible-only-pwa {
  display: none;
  
  @media all and (display-mode: standalone) {
    display: inline;
  }
}
```
I think this could work

**Comment by kevinpapst** [comment]
<img width="573" alt="Bildschirm­foto 2023-04-24 um 20 51 50" src="https://user-images.githubusercontent.com/533162/234088987-1862a470-a83a-40be-bffc-666cbde75735.png">

Lets create the back link like the one in the "password forgotten" screen.
The template is here: https://github.com/kimai/kimai/blob/main/templates/security/unlock.html.twig

Do you want to try a PR? If not, please create a feature request that goes along with the screenshot and your CSS:

**Reply by turanibrahim** [reply]
I can add the button, then create PR about it 👍🏼
